docs/src/windows.md

```diff
@@ -23,47 +22,37 @@ libreadline-dev tk tk-dev
 curl https://pyenv.run | bash
 ```
 
-- Edit ~/.bashrc (ex: ```nano ~/.bashrc```) and add this code at the end (replace {USERNAME} with your Windows session name):
+- Edit ~/.bashrc (ex: ```nano ~/.bashrc```) and add this code at the end (replace {USERNAME} with your Ubuntu subsystem's session name):
 
 ```
 export PATH="/home/{USERNAME}/.pyenv/bin:$PATH"
 eval "$(pyenv init -)"
 eval "$(pyenv virtualenv-init -)"
 ```
 
+- Use updated .bashrc and install webpack
+```
+source ~/.bashrc
+npm i webpack -g
```

Comment:
Why do you need webpack(-cli) as global installs?
They are installed locally in Erdapfel by npm, this should be enough.